projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,41 @@
+// Firmware 102 Homework
+// Vaaranan Yogalingam
+// 2021-05-29
+
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define PERIOD 500
+
+typedef struct counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void increment_callback(SoftTimerId timer_id, void *context) {
```

Comment:
We should add a static keyword and add a 'prv_' prefix to the name of the function to indicate that this is a private function which should only be used by this source file